ntt/naive/ntt.expect

```diff
@@ -0,0 +1,3 @@
+{
+  "cycles": 65241
```

Comment:
Wow, that's a lot of cycles. On your machine, how much time (in seconds) did it take to run this?

Reply:
~1 minute